LEAF_Request_Portal/sources/FormEditor.php

```diff
@@ -152,6 +154,14 @@ function setRequired($indicatorID, $input) {
     								SET required=:input
     								WHERE indicatorID=:indicatorID', $vars);
     }
+
+    function setSensitive($indicatorID, $input) {
+        $vars = array(':indicatorID' => $indicatorID,
+            ':input' => $input);
```

Comment:
Since we know that the `$input` variable needs to be an integer, there should be some sort of type checking here.

http://php.net/manual/en/function.is-int.php